src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleControl.cs

```diff
@@ -13,6 +13,7 @@
 #pragma warning disable 1634, 1691
 
 using System;
+using System.Buffers;
```

Comment:
We are not using `ArrayPool`, so do we still need this?

Reply:
I started with ArrayPool and then migrate to stackalloc.

Removed.